tests/configs.py

```diff
@@ -0,0 +1,236 @@
+        fp_collect_info_iframe_chrome (return 'True' if value is 'object')
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+generateMagicArraysConfig = ScriptConfig(
+    name="generate_magic_arrays_test",
+    script=from_file("generate.magic.arrays.js"),
+    query="""
+    {
+        null
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+iFrameContentWindowConfig = ScriptConfig(
+    name="iframe_contentWindow_test",
+    script=from_file("iframe.contentWindow.js"),
+    query="""
+    {
+        null
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+mediaCodecsConfig = ScriptConfig(
+    name="media_codecs_test",
+    script=from_file("media.codecs.js"),
+    query="""
+    {
+        null
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+navigatorHardWareConcurrencyConfig = ScriptConfig(
+    name="navigator_hardwareConcurrency_test",
+    script=from_file("navigator.hardwareConcurrency.js"),
+    query="""
+    {
+        null
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+navigatorLanguagesConfig = ScriptConfig(
+    name="navigator_languages_test",
+    script=from_file("navigator.languages.js"),
+    query="""
+    {
+        null
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+navigatorPermissionsConfig = ScriptConfig(
+    name="navigator_permissions_test",
+    script=from_file("navigator.permissions.js"),
+    query="""
+    {
+        null
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+navigatorPluginsConfig = ScriptConfig(
+    name="navigator_plugins_test",
+    script=from_file("navigator.plugins.js"),
+    query="""
+    {
+        null
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+navigatorUserAgentConfig = ScriptConfig(
+    name="navigator_userAgent_test",
+    script=from_file("navigator.userAgent.js"),
+    query="""
+    {
+        phantom_ua_test_result (return 'True' if test result is 'ok')
+        headchr_ua_test_result (return 'True' if test result is 'ok')
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+navigatorVendorConfig = ScriptConfig(
+    name="navigator_vendor_test",
+    script=from_file("navigator.vendor.js"),
+    query="""
+    {
+        headchr_vendor_test_result (return 'True' if test result is 'ok')
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+navigatorWebDriverConfig = ScriptConfig(
+    name="navigator_webdriver_test",
+    script=from_file("navigator.webdriver.js"),
+    query="""
+    {
+        webdriver_new_test_result (return 'True' if test result is 'passed')
+        webdriver_advanced_test_result (return 'True' if test result is 'passed')
+        headchr_chrome_obj_test_result (return 'True' if test result is 'ok')
+    }
+    """,
+    url="https://bot.sannysoft.com/",
+)
+
+webGLVendorConfig = ScriptConfig(
+    name="webGL_vendor_test",
+    script=from_file("webgl.vendor.js"),
+    query="""
+    {
+        null
```

Comment:
huh?

Reply:
These are null to represent a placeholder. Was running into malformed query errors so this was just a quick fix for that.
But aside from that, to my knowledge, these scripts do not work as of now and we need to find a better way to test the effectiveness of the script (better website / corresponding query). 

Will leave a comment here to express this.